gitman/git.py

```diff
@@ -15,51 +17,67 @@
 def git(*args, **kwargs):
     return call('git', *args, **kwargs)
 
+def gitsvn(*args, **kwargs):
+    return call('git', 'svn', *args, **kwargs)
+
+def clean_dir(path):    
```

Comment:
This could be implemented as another case in the shell utility.

`shell.rm(path, _keep=True)`